### ISSUE TITLE
Shoot program feed orientation

### ISSUE THREAD

**Issue by duhring** [goodtohear/BeatSheetStudio#55]
Shoot shows an upside down image if the camera is repositioned with the camera to the right rather than to the left.

When I flip the Shoot camera so I can more naturally focus on the lens, I am presented with an upside-down view of the Program Feed.

https://youtu.be/2t2nEcsJq6o

**Comment by michaelforrest** [comment]
Yep. I still haven't decided if this will become an official feature yet because of issues like this!

**Comment by michaelforrest** [comment]
Meanwhile the workaround is clear ;-)
